bbb-common-web/src/main/java/org/bigbluebutton/api/model/request/EndMeeting.java

```diff
@@ -32,7 +33,7 @@
     private String password;

```

Comment:
Does this need to be a `Password`?

Reply:
That is the password the user passed to the API. It is not the same as the Password class that is used for validation.

Reply:
I have them separate since passwords for different requests may have different validation rules.